### ISSUE TITLE
GVMAnalysis

### ISSUE THREAD

**Issue by hwhxy** [sfahl/mallodroid#8]
No module named ganalysis, 
https://androguard.readthedocs.io/en/latest/search.html?q=GVMAnalysis&check_keywords=yes&area=default can't solve this problem         
！！ orz！！

**Comment by luckenzo** [comment]
Yes, that module doesn't exist in newer versions of Androguard (mallodroid depends on it). I fixed this and other deprecated modules in my forked version (https://github.com/luckenzo/mallodroid) sometime in the past.
